src/azure-cli/azure/cli/command_modules/rdbms/_flexible_server_util.py

```diff
@@ -169,7 +169,7 @@ def get_postgres_tiers(sku_info):
 
 def get_postgres_list_skus_info(cmd, location):
     list_skus_client = cf_postgres_flexible_location_capabilities(cmd.cli_ctx, '_')
-    list_skus_result = list_skus_client.execute(location)
+    list_skus_result = list_skus_client.execute(location, api_version='2021-06-01')
```

Comment:
I noticed that in many places you overwrite the `api_version` from SDK default. Why are you doing this? Why not release a new SDK for `2021-06-01` API version?

Reply:
I saw your PR description:
> These operations don't work yet for api-version 2022-01-20-preview: checkNameAvailability, locations/capabilities and getPrivateDnsZoneSuffix. so temporarily we need to force previous version 2021-06-01 for those endpoints

If `2022-01-20-preview` is not ready for all features, can you consider to add new features in cli extension? I remember that there's a extension for rdbms.

Reply:
Because backup operations are only in new api-version 2022-01-20-preview; and as I mentioned in the description, there are only 3 operations that don't work with that new api, so I force them to version 2021-06-01.

Reply:
Generally we can't accept features with such quality to be merged and released in main CLI.

Maybe you should discuss internally whether to wait until the new api version ready or support backup in cli extension.

If you think `rdbms-connect` doesn't match, you can choose to create a general rdbms extension

Reply:
@evelyn-ys `2022-01-20-preview` is now ready, I've removed all references to previous api-versions.